lib/ex_unit/test/ex_unit/diff_test.exs

```diff
@@ -126,10 +126,11 @@ defmodule ExUnit.DiffTest do
     assert env_binding == expected_binding
   end
 
+  @terminal_width 80
```

Comment:
this changes other tests. I can either adjust it or make a custom helper to test it ???